node/index.ts

```diff
@@ -1,27 +1,25 @@
 import { IOClients, Service, ServiceContext } from '@vtex/api'
-import addProductTranslationsCSV from './resolvers/addProductTranslationsCSV'
+import addProductTranslations from './resolvers/addProductTranslations'
 
 const SMALL_TIMEOUT_MS = 500
 
 declare global {
   type Context = ServiceContext<IOClients>
 }
 
-export default new Service({
+export default new Service<IOClients>({
   clients: {
     options: {
       messagesGraphQL: {
-        concurrency: 50,
-        retries: 2,
+        concurrency: 20,
+        retries: 3,
```

Comment:
[⛏] `timeout` is defined using a variable, but `concurrency` and `retries` are plain numbers. Maybe we should have a standard here!

Reply:
`timeout` is defined in a variable to annotate information about its unit.
Since the other ones are unitless, I think using variables doesn't add much.
Buuuut I'll use them for the sake of standardisation.